assignments/assignments_test.go

```diff
@@ -15,7 +15,7 @@ import (
 
 func TestFetchAssignments(t *testing.T) {
 	qfTestOrg := scm.GetTestOrganization(t)
-	s := scm.GetTestSCM(t)
+	s, _ := scm.GetTestUserAndSCM(t)
```

Comment:
Suggest to keep the name `GetTestSCM`, since that's the main purpose. Alternatively, the name should at least mirror the return order: `GetTestSCMAndUser`, but I think it is not necessary...

Reply:
Renamed to `GetTestSCM`.